frontend/src/metabase/databases/components/DatabaseForm/DatabaseForm.tsx

```diff
@@ -34,18 +35,18 @@ export interface DatabaseFormProps {
   setIsDirty?: (isDirty: boolean) => void;
 }
 
-const DatabaseForm = ({
-  engines,
+export const DatabaseForm = ({
   initialValues: initialData,
   autofocusFieldName,
-  isHosted = false,
   isAdvanced = false,
-  isCachingEnabled = false,
```

Comment:
I assume you need to remove both `isHosted` and `isCachingEnabled` from `DatabaseFormProps` as well.

Reply:
Fixed